src/base/dof_map.C

```diff
@@ -889,7 +889,7 @@ void DofMap::clear()
 #ifdef LIBMESH_ENABLE_AMR
 
   _dof_constraints.clear();
-  _stashed_dof_constraints.clear();
+  //_stashed_dof_constraints.clear();
```

Comment:
Surely we at least want to clear stashed constraints when we're clearing the whole DofMap, no?

Reply:
Yeah I think that one can go back in :)